client/src/app/components/header/header.component.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import {
+  MATERIAL_MODULE_DEPENDENCIES,
+  NGRX_STORE_MODULE,
```

Comment:
Hey since we don't use NGRX_STORE_MODULE we can remove it. Sonarcloud picked it up as a code smell